.circleci/config.yml

```diff
@@ -454,6 +454,25 @@ jobs:
             make test
             cd ../..
 
+  end2end-test-bridge-validator:
+    executor: ubuntu-builder
+    steps:
+      - attach_workspace:
+          at: '~'
+      - config-bridge-validator-path
+      - run:
+          name: Install parity
+          command: curl -o ~/bin/parity https://releases.parity.io/ethereum/v2.3.6/x86_64-unknown-linux-gnu/parity
+      - run:
+          name: Make parity executable
+          command: chmod 755 ~/bin/parity
+      - run:
+          name: Run pytest
+          command: |
+            cd tools/bridge
+            make test-end2end
+            cd ../..
+
```

Comment:
This could be written as `make -C tools/bridge test-end2end`

Reply:
Ah okay, I copied it from the other circle ci code. I changed it everywhere.